metadata-ingestion/src/datahub/ingestion/source/vertexai.py

```diff
@@ -0,0 +1,715 @@
+                        ),
+                        customProperties={"displayName": endpoint.display_name},
+                    )
+                )
+
+                # TODO add followings when metadata for MLModelDeployment is updated (these aspects not supported currently)
+                # aspects.append(
+                #     ContainerClass(container=self._get_project_container().as_urn())
+                # )
+                # aspects.append(SubTypesClass(typeNames=[MLTypes.ENDPOINT]))
+
+                yield from MetadataChangeProposalWrapper.construct_many(
+                    endpoint_urn, aspects=aspects
+                )
+
+    def _gen_ml_model_mcps(
+        self, ModelMetadata: ModelMetadata
+    ) -> Iterable[MetadataChangeProposalWrapper]:
+        """
+        Generate an MLModel and Endpoint mcp for an VertexAI Model Version.
+        """
+
+        model: Model = ModelMetadata.model
+        model_version: VersionInfo = ModelMetadata.model_version
+        training_job_urn: Optional[str] = ModelMetadata.training_job_urn
+        endpoints: Optional[List[Endpoint]] = ModelMetadata.endpoints
+        endpoint_urns: List[str] = list()
+
+        logging.info(f"generating model mcp for {model.name}")
+
+        # Generate list of endpoint URL
+        if endpoints:
+            for endpoint in endpoints:
+                logger.info(
+                    f"found endpoint ({endpoint.display_name}) for model ({model.resource_name})"
+                )
+                endpoint_urns.append(
+                    builder.make_ml_model_deployment_urn(
+                        platform=self.platform,
+                        deployment_name=self._make_vertexai_endpoint_name(
+                            entity_id=endpoint.display_name
+                        ),
+                        env=self.config.env,
+                    )
+                )
+
+        # Create URN for Model and Model Version
+        model_group_urn = self._make_ml_model_group_urn(model)
+        model_name = self._make_vertexai_model_name(entity_id=model.name)
+        model_version_name = f"{model_name}_{model_version.version_id}"
+        model_urn = self._make_ml_model_urn(model_version, model_name=model_name)
+
+        # Create aspects for ML Model
+        aspects: List[_Aspect] = list()
+
+        aspects.append(
+            MLModelPropertiesClass(
+                name=model_version_name,
+                description=model_version.version_description,
+                customProperties={
+                    "displayName": f"{model_version.model_display_name}",
+                    "versionId": f"{model_version.version_id}",
+                    "resourceName": model.resource_name,
+                },
+                created=TimeStampClass(
+                    datetime_to_ts_millis(model_version.version_create_time)
+                )
+                if model_version.version_create_time
+                else None,
```

Comment:
some parens around this would make it a bit more readable (after the auto-formatter runs)
```suggestion
                created=(
                	TimeStampClass(
	                    datetime_to_ts_millis(model_version.version_create_time)
	                )
	                if model_version.version_create_time
	                else None
                ),
```

Reply:
lintFix automatically change the format this way,   should we fix rule?    @hsheth2

Reply:
yeah this trick works with lintfix,  i changed other part like this